SMLHelper/Patchers/EnumPatching/TechCategoryPatcher.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace SMLHelper.V2.Patchers.EnumPatching
+{
+    using System;
+    using System.Collections.Generic;
+    using SMLHelper.V2.Handlers;
+    using Utility;
+
+    internal class TechCategoryPatcher
+    {
+        private const string TechCategoryEnumName = "TechCategory";
+
+        internal const int startingIndex = 0; // The default TechCategory contains indexes 0 through 14
```

Comment:
this looks a bit odd to me since the TechCategory's last index is 25 while u assigned it to 0 so "anything below 0 is a vanilla TechCategory" in that case.